shared-module/rgbmatrix/RGBMatrix.c

```diff
@@ -84,31 +84,36 @@ void common_hal_rgbmatrix_rgbmatrix_reconstruct(rgbmatrix_rgbmatrix_obj_t* self,
         _PM_FREE(self->core.screenData);
```

Comment:
Why does the framebuffer case above call mp_obj_new_bytearray? Isn't the idea that the framebuf to use is provided?

Reply:
Weird.  This line of preexisting code should probably also be deleted.